aws-code-pipeline/variables.tf

```diff
@@ -0,0 +1,81 @@
+variable "AWS_ACCESS_KEY_ID" {
+  type    = string
+  default = "AWS_ACCESS_KEY_ID"
+}
+
+variable "AWS_SECRET_ACCESS_KEY" {
+  type    = string
+  default = "AWS_SECRET_ACCESS_KEY"
+}
+
+variable "aws_region" {
+  type        = string
+  description = "AWS Region"
+  default     = "us-west-2"
+}
+
+variable "aws_account_id" {
+  type        = string
+  description = "AWS Account ID"
+  default     = "12345678910"
+}
+
+variable "s3_bucket_name" {
+  type        = string
+  description = "S3 bucket to store pipeline artifacts"
+  default     = "s3-bucket-name"
+}
+
+variable "code_pipeline_name" {
+  type        = string
+  description = "Name for AWS CodePipeline"
+  default     = "mage-codepipeline"
+}
+
+variable "ecr_repo_name" {
+  type        = string
+  description = "ECR repo name"
+  default     = "ecr-repo-name"
+}
+
+variable "ecr_image_tag" {
```

Comment:
Are you able to use github SHA as image tag so that the docker images are versioned?

Reply:
i can't, the ecr source can only look at one tag